lib/Player.js

```diff
@@ -6,14 +6,13 @@ class Player {
   
   increaseScore(score){
     this.score += score;
-    domUpdates.changeScoreboard(this.score);
+    domUpdates.changeScoreboard(score);
   } 
  
   decreaseScore(score) {
     this.score -= score;
-    domUpdates.changeScoreboard(this.score);
+    domUpdates.changeScoreboard(score);
```

Comment:
Don't want to merge these to master, these are old changes that we updated yesterday I believe.

Reply:
Agree, wont merge this!